files/qtip2/update.json

```diff
@@ -1,8 +1,9 @@
 {
-  "packageManager": "github",
+  "packageManager": "npm",
   "name": "qtip2",
-  "repo": "qtip2/bower",
+  "repo": "qtip2/qtip2",
   "files": {
-    "include": ["*.js", "*.css", "*.map", "./basic/**/*"]
+    "basePath": "dist",
+    "include": ["**/*.js", "**/*.css", "**/*.map"]
```

Comment:
remove the include line completely to add everything inside dist/ by default.
No need to rebase or anything